packages/nl-design-system-unstable/__docs__/content-richtlijnen.md

```diff
@@ -22,42 +22,39 @@ Bv. help bestanden, veelgestelde vragen, contactgegevens.
 
 Bv. knoptekst, formulierlabels, feedbacktekst, contextuele helptekst.
 
-## Toegankelijkheid
+## Voor wie schrijven we
 
-Taal en toegankelijkheid gaan hand in hand. Als de tekstuele inhoud van een product niet of moeilijk begrepen wordt, dan is het product niet toegankelijk.
+Taal en toegankelijkheid gaan hand in hand. Als de tekstuele inhoud van een product niet of moeilijk begrepen wordt, dan is het product niet toegankelijk. Duidelijke taal zorgt er voor dat iedereen begrijpt wat je met je tekst wilt overbrengen.
 
-Duidelijke taal zorgt er voor dat iedereen begrijpt wat je met je tekst wilt overbrengen.
+Veel mensen hebben baat bij duidelijke taal:
 
-### Laaggeletterdheid
+### Mensen die laaggeletterd zijn
 
-Laaggeletterdheid is een term voor mensen die grote moeite hebben met lezen, schrijven en/of rekenen. Mensen die laaggeletterd zijn, zijn geen analfabeten. Ze kunnen wel lezen en schrijven, maar beheersen niet het eindniveau VMBO of niveau MBO-2/3.
+Mensen die laaggeletterd hebben moeite met lezen, schrijven en/of rekenen. Verwar dit niet met de term  analfabeten. Ze kunnen wel lezen en schrijven, maar beheersen niet het eindniveau vmbo of niveau mbo-2/3.
 
-18% van de Nederlandse bevolking heeft moeite met lezen en schrijven. Dat zijn 3,1 miljoen mensen. Hiervan heeft 2 miljoen Nederlands als eerste taal en 1,1 miljoen als tweede taal.
+### Mensen die Nederlands niet als moedertaal hebben
 
-Mensen die laaggeletterd zijn hebben onder meer moeite met:
+Het kunnen mensen zijn die doof zijn en waarbij gebarentaal hun moedertaal is. Gebarentaal heeft een andere grammatica en vocabulaire. Het zijn ook mensen met een migratieachtergrond en die een andere moedertaal hebben dan het Nederlands.
```

Comment:
"gebarentaal → "Nederlandse Gebarentaal (NGT)"?

Reply:
Changed this!